simple/app.go

```diff
@@ -500,14 +503,27 @@ func (a *App) Convert(_ context.Context, req app.ConversionRequest) (*app.RawObj
 	}, err
 }
 
-// CallResourceCustomRoute implements app.App and handles custom resource route requests
-func (a *App) CallResourceCustomRoute(ctx context.Context, req *app.ResourceCustomRouteRequest) (*app.ResourceCustomRouteResponse, error) {
-	k, ok := a.kinds[gvk(req.ResourceIdentifier.Group, req.ResourceIdentifier.Version, req.ResourceIdentifier.Kind)]
+// CallCustomRoute implements app.App and handles custom resource route requests
+func (a *App) CallCustomRoute(ctx context.Context, req *app.CustomRouteRequest) (*app.CustomRouteResponse, error) {
+	if req.ResourceIdentifier.Kind == "" && req.ResourceIdentifier.Plural == "" {
+		scope := resource.NamespacedScope
+		if req.ResourceIdentifier.Namespace == "" {
+			scope = resource.ClusterScope
+		}
+		if handler, ok := a.customRoutes[a.customRouteHandlerKey(nil, req.Method, req.Path, scope)]; ok {
```

Comment:
do resource level custom routes also need to be added to `AppConfig`?

Reply:
Yeah, this PR doesn't aim to support them yet in `simple.App`, but we'll need a way to add them, which would have to be at the `AppConfig` level.